### PR TITLE
feat: Fixed up feature previews panel

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

The feedback popover was opening the panel, but wasn't in there!

## Changes

* Only open the panel if the chosen state is actually supported
* Moved the feature previews to the panel
* Tidied the styles for the feedback submission
* Also fixes the overflow for docs panel

Screenshot 2023-11-28 at 14 18 14

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/layout/FeaturePreviews/FeaturePreviewsModal.tsx [hunk 1]
```diff
@@ -119,19 +126,31 @@ function FeaturePreview({ feature }: { feature: EnrichedEarlyAccessFeature }): J
...
-                    </LemonButton>
+                            }}
+                        >
+                            Cancel
+                        </LemonButton>
+
+                        <LemonButton
+                            type="primary"
+                            onClick={() => {
+                                void submitEarlyAccessFeatureFeedback(feedback).then(() => {
```
Comment (daibhin) [inline frontend/src/layout/FeaturePreviews/FeaturePreviewsModal.tsx:143]: Any reason not to use an `async` function here?

**Review by daibhin — APPROVED** [review]
Not feedback for this PR but noticed two things in the screenshot on the description:
1. I think you removed the orange active state on text inputs so they didn't look like error states. Wonder if we should do the same for the text areas?
2. The two oranges in the switches & the buttons are very different. The buttons are a lot more yellow.

**Comment by Twixes** [comment]
Looks good! Although kind of dry without the "Get early access to these upcoming features. Let us know what you think!" intro IMO. Can we add it back?

**Comment by benjackwhite** [comment]
> Looks good! Although kind of dry without the intro ("Get early access to these upcoming features. Let us know what you think!") IMO. Can we add it to the side panel version too?

Ah yeah I knew I forgot something...

File: frontend/src/layout/FeaturePreviews/FeaturePreviewsModal.tsx (hunk 1 above)
Comment (benjackwhite) [reply frontend/src/layout/FeaturePreviews/FeaturePreviewsModal.tsx:143]: It's not allowed as the onClick doesn't take a promise

Comment (benjackwhite) [reply frontend/src/layout/FeaturePreviews/FeaturePreviewsModal.tsx:143]: Since the eslint changes that is

**Comment by benjackwhite** [comment]
> 1. I think you removed the orange active state on text inputs so they didn't look like error states. Wonder if we should do the same for the text areas?

Fixed that 

> 2. The two oranges in the switches & the buttons are very different. The buttons are a lot more yellow.

Yeah...

**Pull request merged**